### PR TITLE
feat(fe): Un/Follow User and List Followers and Followings

### LINKED ISSUE #371: Un/Follow User and List Followers and Followings
Users could be un/followed on their profiles.
User can list their followers and followings.

### PR DESCRIPTION
**Pull request by emindeniz99**
Users could be un/followed on their profiles.
User can list their followers and followings.

close #371

### CODE REVIEW

**Comment by emindeniz99** [comment]
click stats
image

list:
image


unfollow:
image


spinner:

image

follow:
image

File: source/frontend/src/components/Profile/index.tsx [hunk 1]
```diff
@@ -170,6 +228,41 @@ const Profile = (props: Props) => {
             </div>
           </div>
         </div>
+        {username && (
```
Comment (meltemarsl) [inline source/frontend/src/components/Profile/index.tsx:231]: The button should not be transparent, color should be given -referably different colors for different states-.

Comment (emindeniz99) [reply source/frontend/src/components/Profile/index.tsx:231]: image
image

**Comment by meltemarsl** [comment]
The adjustment looks fine. Since there is no functional problem, I am merging it.

**Pull request merged**